bundles/org.openhab.core.config.discovery/src/main/resources/OH-INF/i18n/inbox.properties

```diff
@@ -0,0 +1,7 @@
+config.system.inbox.autoIgnore.label = Auto Ignore
+config.system.inbox.autoIgnore.description = <![CDATA[
+                <p>If enabled, an inbox result matching an existing thing is automatically ignored.</p>
+                <p>If set to false, the inbox may contain results although identical things already exist.</p>
+            ]]>
```

Comment:
It doesn't work for me. I tested with the following `inbox_nl.properties`:

```
system.config.inbox.autoIgnore.label = NL_Auto Ignore
system.config.inbox.autoIgnore.description = <![CDATA[
                <p>NL_If enabled, an inbox result matching an existing thing is automatically ignored.</p>
                <p>NL_If set to false, the inbox may contain results although identical things already exist.</p>
            ]]>
system.config.inbox.autoApprove.label = NL_Auto Approve
system.config.inbox.autoApprove.description = NL_If enabled, inbox results are automatically approved, unless they were marked as ignored.
````

This is what it looks like in the UI:

![Screenshot from 2021-04-27 16-59-53](https://user-images.githubusercontent.com/12213581/116264006-34e1e400-a77a-11eb-9220-e8a646a03b6c.png)

It ignores the content when it is not on the same line, see the REST API response JSON:

```json
{
   "uri":"system:inbox",
   "parameters":[
      {
         "defaultValue":"true",
         "description":"\u003c![CDATA[",
         "label":"NL_Auto Ignore",
         "name":"autoIgnore",
         "required":false,
         "type":"BOOLEAN",
         "readOnly":false,
         "multiple":false,
         "advanced":false,
         "verify":false,
         "limitToOptions":true,
         "options":[
            
         ],
         "filterCriteria":[
            
         ]
      },
      {
         "defaultValue":"false",
         "description":"NL_If enabled, inbox results are automatically approved, unless they were marked as ignored.",
         "label":"NL_Auto Approve",
         "name":"autoApprove",
         "required":false,
         "type":"BOOLEAN",
         "readOnly":false,
         "multiple":false,
         "advanced":false,
         "verify":false,
         "limitToOptions":true,
         "options":[
            
         ],
         "filterCriteria":[
            
         ]
      }
   ],
   "parameterGroups":[
      
   ]
}
```

So it will probably also not work with Crowdin. Besides that it is also not easy to translate strings that contain all this markup. Is the HTML still required for the UI @ghys?

Reply:
Thank you both for testing and the proposals. I will remove the Character Data and replace HTML paragraph elements by line breaks.